akka-cluster/src/test/scala/akka/cluster/JoinConfigCompatPreDefinedChecksSpec.scala

```diff
@@ -87,56 +89,62 @@ class JoinConfigCompatPreDefinedChecksSpec extends WordSpec with Matchers {
+      JoinConfigCompatChecker.fullMatch(
+        requiredKeys,
+        config("""
+            |{
+            | akka.cluster.min-nr-of-members = 1
+            |}
+          """.stripMargin),
+        clusterConfig) match {
+        case Valid =>
+          fail()
+        case Invalid(incompatibleKeys) =>
+          incompatibleKeys should have size 2
+          incompatibleKeys should contain("akka.cluster.retry-unsuccessful-join-after is missing")
+          incompatibleKeys should contain("akka.cluster.allow-weakly-up-members is missing")
+      }
     }
 
     "fail when all required keys are passed, but some values don't match cluster config" in {
 
-      val Invalid(incompatibleKeys) =
-        JoinConfigCompatChecker.fullMatch(
-          requiredKeys,
-          config("""
-              |{
-              | akka.cluster.min-nr-of-members = 1
-              | akka.cluster.retry-unsuccessful-join-after = 15s
-              | akka.cluster.allow-weakly-up-members = off
-              |}
-            """.stripMargin),
-          clusterConfig)
-
-      incompatibleKeys should have size 2
-      incompatibleKeys should contain("akka.cluster.retry-unsuccessful-join-after is incompatible")
-      incompatibleKeys should contain("akka.cluster.allow-weakly-up-members is incompatible")
+      JoinConfigCompatChecker.fullMatch(
+        requiredKeys,
+        config("""
+            |{
+            | akka.cluster.min-nr-of-members = 1
+            | akka.cluster.retry-unsuccessful-join-after = 15s
+            | akka.cluster.allow-weakly-up-members = off
+            |}
+          """.stripMargin),
+        clusterConfig) match {
+        case Valid =>
+          fail()
+        case Invalid(incompatibleKeys) =>
+          incompatibleKeys should have size 2
+          incompatibleKeys should contain("akka.cluster.retry-unsuccessful-join-after is incompatible")
+          incompatibleKeys should contain("akka.cluster.allow-weakly-up-members is incompatible")
+      }
     }
 
     "fail when all required keys are passed, but some are missing and others don't match cluster config" in {
 
-      val Invalid(incompatibleKeys) =
-        JoinConfigCompatChecker.fullMatch(
-          requiredKeys,
-          config("""
-              |{
-              | akka.cluster.min-nr-of-members = 1
-              | akka.cluster.allow-weakly-up-members = off
-              |}
-            """.stripMargin),
-          clusterConfig)
-
-      incompatibleKeys should have size 2
-      incompatibleKeys should contain("akka.cluster.retry-unsuccessful-join-after is missing")
-      incompatibleKeys should contain("akka.cluster.allow-weakly-up-members is incompatible")
+      JoinConfigCompatChecker.fullMatch(
+        requiredKeys,
+        config("""
+            |{
+            | akka.cluster.min-nr-of-members = 1
+            | akka.cluster.allow-weakly-up-members = off
+            |}
+          """.stripMargin),
```

Comment:
It seems like all of these done this way with `{..}` should be
```scala
config("""
   akka.cluster.min-nr-of-members = 1
   akka.cluster.retry-unsuccessful-join-after = 10s
    akka.cluster.allow-weakly-up-members = on
  """.stripMargin)
```

Reply:
I agree, but perhaps let's change that in another PR (the weird formatting wasn't introduced here)